docs/usage.md

```diff
@@ -274,6 +240,46 @@ params {
 }
 ```
 
+### Updating containers
+
+The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. If for some reason you need to use a different version of a particular tool with the pipeline then you just need to identify the `process` name and override the Nextflow `container` definition for that process using the `withName` declaration. For example, in the [nf-core/viralrecon](https://nf-co.re/viralrecon) pipeline a tool called [Pangolin](https://github.com/cov-lineages/pangolin) has been used during the COVID-19 pandemic to assign lineages to SARS-CoV-2 genome sequenced samples. Given that the lineage assignments change quite frequently it doesn't make sense to re-release the nf-core/viralrecon everytime a new version of Pangolin has been released. However, you can override the default container used by the pipeline by creating a custom config file and passing it as a command-line argument via `-c custom.config`.
```

Comment:
Shouldn't be changed by an `nf-core/rnaseq` specific module?

Reply:
These will end up being generic docs so in theory could come from any pipeline. We do need to move that at some point because it has come up quite a bit recently...maybe to nf-core/modules have a `docs/` sections there. We can then just point to these from the pipeline documentation in the template.

I was hoping `For example, in the [nf-core/viralrecon](https://nf-co.re/viralrecon) pipeline` would clarify that it is just an example 😅

Reply:
Fair enough! 😄